repoarchivetool/storage_interface.py

```diff
@@ -1,4 +1,58 @@
 import json
+import boto3
+from botocore.exceptions import ClientError
+
+def get_bucket_content(filename: str) -> bool | ClientError:
+    """
+        Downloads a given file from an S3 Bucket
+
+        ==========
+
+        Args:
+            filename (str): The name of the file to download
+
+        Returns:
+            Bool or ClientError
+    """
+
+    # Use this for running locally. Replace <profile_name> with a profile from ~/.aws/credentials
+    # session = boto3.Session(profile_name="<profile_name>")
+
+    # Use this for container running (uses environment credentials instead)
+    session = boto3.Session()
+    s3 = session.client("s3")
+
+    try:
+        s3.download_file("github-audit-tool", f"repo-archive/{filename}", filename)
+    except ClientError as e:
+        return e
+    return True
+
+def update_bucket_content(filename: str) -> bool | ClientError:
+    """
+        Uploads a given file to an S3 Bucket
+
+        ==========
+
+        Args:
+            filename (str): The name of the file to upload
+
+        Returns:
+            Bool or ClientError
+    """
+
+    # Use this for running locally. Replace <profile_name> with a profile from ~/.aws/credentials
```

Comment:
I think it is better to remove this and just mention in the readme that the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY need to be set appropriately when running locally.